### ISSUE TITLE
Using numeric strings as column labels causes problems

### ISSUE THREAD

**Issue by MarcinOrlowski** [MarcinOrlowski/php-ascii-table#22]
Using numeric strings as column labels causes problems:

```php
$table = new TextTable(['1', '2']);
$table->addRow(['A', 'B']);
```

would yield `MarcinOrlowski\TextTable\Exceptions\ColumnKeyNotFoundException: Unknown column key: 0` exception.

**Comment by MarcinOrlowski** [comment]
The problem lies in PHP which turn `"1"` (string) into `1` (int) when such string is used as the table key:

```bash
$ php -r "var_dump(['1' => 'foo']);"                                                                                                                                                           ⌚10:46
array(1) {
[1]=>
string(3) "foo"
}
```

The workaround could be to automatically prefix all the string keys with something to prevent that (i.e. `"1"` into `"str_1"` to prevent that conversion. This can be added in `ColumnContainer`'s `addColumn()` but in the way that will not break i.e. array access implementation etc.

**Comment by MarcinOrlowski** [comment]
The problem lies in PHP which turn `"1"` (string) into `1` (int) when such string is used as the table key:

```bash
$ php -r "var_dump(['1' => 'foo']);"                                                                                                                                                           ⌚10:46
array(1) {
[1]=>
string(3) "foo"
}
```

The workaround could be to automatically prefix all the string keys with something to prevent that (i.e. `"1"` into `"str_1"` to prevent that conversion. This can be added in `ColumnContainer`'s `addColumn()` but in the way that will not break i.e. array access implementation etc.

**Comment by MarcinOrlowski** [comment]
Fixed in v2.5.0
